service/src/main/java/fish/focus/uvms/plugins/ais/service/AisService.java

```diff
@@ -239,4 +258,12 @@ public Set<String> getKnownFishingVessels() {
     public int getKnownFishingVesselsSize() {
         return knownFishingVessels.size();
     }
+
+    void setAssetListOK(boolean assetListOK) {
+        isAssetListOK = assetListOK;
+    }
+
+    boolean isAssetListOK(){
```

Comment:
we don't need a getter method since we're only accessing the field within this class